Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyBoolComboBoxCtrl.hxx

```diff
@@ -52,7 +58,11 @@ namespace AzToolsFramework
         void onChildComboBoxValueChange(int value);
 
     private:
+        virtual void OnEditButtonClicked();
```

Comment:
question: Does this definitely need to be virtual? Are we expecting clients will need to override this and change the behavior?

Reply:
It doesn't need to be no, I thought that the user might use that to override the click logic but it doesn't necessarily work with how everything is set up

Reply:
Yeah I'd just remove it to remove confusion, this seems like the main thing you'd ever really want to do. Thanks!